examples/user_guide/06-Live_Data.ipynb

```diff
@@ -549,8 +534,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts GridSpace [show_legend=True fig_size=200]\n",
-    "waves_dmap.overlay('amplitude').grid('frequency')"
+    "waves_dmap.overlay('amplitude').grid('frequency').opts(fig_size=200, show_legend=True)"
```

Comment:
Too much stuff happening on a line that also includes `.opts`!

Reply:
```suggestion
    "wave_grid = waves_dmap.overlay('amplitude').grid('frequency')\n",
    "wave_grid.opts(fig_size=200, show_legend=True)"
```